### PR TITLE
Updated what is Tizen

### PR DESCRIPTION
**Pull request by omar-srbd**
### Change Description ###

This PR is the continuation of https://github.com/Samsung/tizen-docs/pull/1647
@asm-foysal @mijin85cho
This PR contains changes that could not be implemented in PRs:

- https://github.com/Samsung/tizen-docs/pull/1593
- https://github.com/Samsung/tizen-docs/pull/1597
- https://github.com/Samsung/tizen-docs/pull/1604
- https://github.com/Samsung/tizen-docs/pull/1605
- https://github.com/Samsung/tizen-docs/pull/1606
- https://github.com/Samsung/tizen-docs/pull/1611
- https://github.com/Samsung/tizen-docs/pull/1615
- https://github.com/Samsung/tizen-docs/pull/1616

PR created by: [omar.saeed@samsung.com](mailto:omar.saeed@samsung.com)

### CODE REVIEW

File: docs/platform/what-is-tizen/versions/tizen-6-0-m2.md [hunk 1]
```diff
@@ -37,22 +37,22 @@ Tizen 6.0 provides **multi-device offloading service** technology using multiple
 
 ![img](media/6.0_1_web.png)
 
-## Lightweight Web Solution
+## Lightweight web solution
 
-Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES 10 up to a large extent:
+Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES10 up to a large extent:
```
Comment (safir-srbd) [inline docs/platform/what-is-tizen/versions/tizen-6-0-m2.md:42]:
```suggestion
Escargot is the Tizen JavaScript engine for products having hardware resource restrictions. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES10 up to a large extent:
```

File: docs/platform/what-is-tizen/versions/tizen-6-5-m2.md [hunk 2]
```diff
@@ -90,20 +90,20 @@ providing interactive communication among devices in a local network.
 - **Flutter for Tizen**:
 
   Flutter is one of the most popular cross-platform frameworks, and it can run on many OSes,
-  such as Android, iOS, Windows, Linux and Mac.
+  such as Android, iOS, Windows, Linux, and Mac.
 
   With flutter-tizen, application developers can deploy faster, get native performance,
```
Comment (safir-srbd) [inline docs/platform/what-is-tizen/versions/tizen-6-5-m2.md:95]:
```suggestion
  With Flutter for Tizen, application developers can deploy faster, get native performance,
```

File: docs/platform/what-is-tizen/versions/tizen-6-5-m2.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ providing interactive communication among devices in a local network.
   Tizen image creator creates your own platform image with building blocks
 
   It provides more than 450 pre-built building blocks, you can combine these blocks to build a desired platform quickly.
```
Comment (safir-srbd) [inline docs/platform/what-is-tizen/versions/tizen-6-5-m2.md:127]:
```suggestion
  It provides more than 450 pre-built building blocks, you can combine these blocks to build the desired platform quickly.
```

**Review by safir-srbd — COMMENTED** [review]
@omar-srbd Thank you for making the necessary edits to the What is Tizen section.

I have added a few suggestions on the doc. For the parts that I couldn't directly suggest, here are the changes:

**File: docs/platform/what-is-tizen/versions/tizen-5-0-m2.md**

Line 4: ".....With the result, the end-users will get a better experience with more enhanced applications."
Suggestion: "As a result, the end-users will get a better experience with more enhanced applications."

Line 32: "Tizen 5.0 provides easy watch app development with Tizen Circular UI APIs and also provides Tizen Wearable App project template that eases development of the circular wearable applications."
Suggestion: "Tizen 5.0 provides easy watch app development with Tizen Circular UI APIs and also provides Tizen Wearable App project templates that eases the development of the circular wearable application."

**File: docs/platform/what-is-tizen/versions/tizen-5-5-m2.md**

Line 55:   "ML API sets are based on [NNStreamer](https://github.com/nnsuite/nnstreamer) that allows construction of the GStreamer pipelines that consists of the conventional GStreamer media plugins and the neural network plugins."
Suggestion: "ML API sets are based on [NNStreamer](https://github.com/nnsuite/nnstreamer) that allows the construction of the GStreamer pipelines that consists of the conventional GStreamer media plugins and the neural network plugins."

Line 74: "To build the Tizen IoT device ecosystem. Tizen open-source community worked in collaboration with other partners. With the result, Tizen IoT profile supports open hardware platforms such as RPI3 and SDTA7X2."
Suggestion: "To build the Tizen IoT device ecosystem. Tizen open-source community worked in collaboration with other partners. As a result, Tizen IoT profile supports open hardware platforms such as RPI3 and SDTA7X2."

Line 75: "Several IoT Cloud solutions are also integrated in Tizen 5.5. Tizen 5.5 provides IoT Headless and IoT Headed profile based on the building blocks for supporting various purpose devices."
Suggestion: "Several IoT Cloud solutions are also integrated in Tizen 5.5. Tizen 5.5 provides IoT Headless and IoT Headed profiles based on the building blocks for supporting various purpose devices."

**File: docs/platform/what-is-tizen/versions/tizen-6-5-m2.md**

Line 6: "With enhanced Tizen Image Creator, you can create custom Tizen image with just a few clicks on Tizen 6.5."
Suggestion: "With the enhanced Tizen Image Creator, you can create custom Tizen images with just a few clicks on Tizen 6.5."

File: docs/platform/what-is-tizen/overview.md [hunk 4]
```diff
@@ -3,17 +3,17 @@
...
 
-## Openness, Optimization, and Opportunities
+## Openness, optimization, and opportunities
 
-Tizen is a Linux-based, open-source software platform. Anyone can contribute and freely develop the system software and applications, which is beneficial for both device manufacturers and application developers. Furthermore, by supporting Web applications with open Web technologies, Tizen provides developers flexibility and extensibility. Based on the open ecosystem, the user can expect various experiences from Tizen devices and services. With its openness, Tizen can pursue continuing progress and expand the ecosystem.
+Tizen is a Linux-based, open-source software platform. Anyone can contribute and freely develop the system software and applications, which is beneficial for both device manufacturers and application developers. Furthermore, by supporting web applications with open web technologies, Tizen provides developers flexibility and extensibility. Based on the open ecosystem, the user can expect various experiences from Tizen devices and services. With its openness, Tizen can pursue continuing progress and expand the ecosystem.
```
[the remark this answers is not in the document]
Comment (safir-srbd) [reply docs/platform/what-is-tizen/overview.md:16]: @omar-srbd You missed this.

**Review by mijin85cho — COMMENTED** [review]
I really appreciate your checking & update.

I also checked the updates, and have below questions.
Would you please check?

File: docs/platform/what-is-tizen/versions/tizen-6-5-m2.md [hunk 5]
```diff
@@ -44,10 +44,10 @@ providing interactive communication among devices in a local network.
 
   Application offloading service framework remotely executes APIs and operations using in-home edge device
 
-  Application offloading Frameworks includes computation and resource offloading.
+  Application offloading frameworks includes computation and resource offloading.
 
-    - With computation offloading, calculation-intensive code blocks can be executed on a remote device and as a result, it achieves faster response time.
-    - With resource offloading, even if your application requires a resource not available on your device. For features that require hardware support, automatically find and match device, and remotely execute required functions.
+    - With computation offloading, calculation-intensive code blocks can be executed on a remote device and as a result, it achieves a faster response time.
+    - With resource offloading, even if your application requires a resource not available on your device. For features that require hardware support, automatically find and match devices, and remotely execute required functions.
```
[the remark this answers is not in the document]
Comment (omar-srbd) [reply docs/platform/what-is-tizen/versions/tizen-6-5-m2.md:50]: Hello @mijin85cho, I hope you are doing well.

I see that the sentence is incomplete but I am unsure what the missing part should contain.
Can you please help me with this matter?

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/platform/what-is-tizen/versions/tizen-6-5-m2.md:50]: > Hello @mijin85cho, I hope you are doing well.
> 
> I see that the sentence is incomplete but I am unsure what the missing part should contain. Can you please help me with this matter?

It seems during the modification the incomplete sentence was left. 
In the [history of this update](https://github.com/Samsung/tizen-docs/commit/6e1e2618b332dcce968cc4dcd2a8d2f903d991e4), the previous sentence was : 
`For features that require hardware support, automatically find and match device, and remotely execute required functions. => Provide a simple way to share the resources through seamless connections between devices`

So you can trim the sentence. Maybe you can trim it as "With resource offloading, automatically find and match devices, and remotely execute required functions." or "With resource offloading, even if your application requires a resource not available on your device, provide a simple way to share the resources through seamless connections between devices. For features that require hardware support, automatically find and match devices, and remotely execute required functions." or something else.

**Review by mijin85cho — COMMENTED** [review]
About my queries, I checked that they were fixed.
If the comments from other SRBD reviewer are all applied, I think  you can proceed to next step.

**Pull request merged**